src/docs-website/docs/api/for-cities/recent-measurements.md

```
@@ -0,0 +1,172 @@
+
+Retrieve the latest hourly calibrated air quality measurements from all public monitoring sites within your city's Grid. This endpoint returns data from approximately the last 7 days.
+
+:::info Tier requirement
+Available on all tiers including Free.
+:::
+
+---
+
+## Endpoint
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)